pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -205,7 +208,7 @@ func (a *Bootstrap) addStorageFiles(base string, uri string, templateData *boots
 	} else {
 		mode = 0600
 	}
-	ign := ignition.FileFromBytes(strings.TrimSuffix(base, ".template"), mode, data)
+	ign := ignition.FileFromBytes(strings.TrimSuffix(base, ".template"), "root", mode, data)
 	if filename == ".bash_history" {
 		ign.User = &igntypes.NodeUser{Name: ignitionUser}
```

Comment:
You could set a variable for the user name and override for `.bash_history` before feeding into `FileFromBytes`.  But certainly not a blocker; we can always do something like that later if we want ;).